packages/react/src/components/Tabs/TabList.tsx

```diff
@@ -19,8 +19,8 @@ import { TabsContext } from './TabsRoot';
 export const TabList = forwardRef<
   HTMLDivElement,
   HTMLAttributes<HTMLDivElement>
->(({ children, className, ...rest }, ref) => {
-  const { value } = useContext(TabsContext);
+>(function TabList({ children, className, ...rest }, ref) {
+  const { value } = useContext(Context);
```

Comment:
Suggest `TabsList`

Reply:
good point <3 was just following the previous code actually, but it should be consistent so I agree changing 🙌